src/Fixer/Basic/BracesFixer.php

```diff
@@ -661,7 +661,7 @@ private function fixMissingControlBraces(Tokens $tokens): void
 
             // do not add for short 'if' followed by alternative loop,
             // for example: if ($a) while ($b): ? > X < ?php endwhile; ? >
-            if ($tokenAfterParenthesis->isGivenKind([T_FOR, T_FOREACH, T_SWITCH, T_WHILE])) {
+            if ($tokenAfterParenthesis->isGivenKind([T_FOR, T_FOREACH, T_SWITCH, T_WHILE, T_IF])) {
```

Comment:
Are you sure it isn't against comment at line 662?

I am definitely not familiar with the codebase, maybe just misunderstood those comment.

Reply:
thanks, I've updated the comment as it indeed didn't cover all the cases that are handled now :+1: